packages/react-native/src/Authenticator/common/DefaultContent/styles.ts

```diff
@@ -0,0 +1,33 @@
+import { StyleSheet } from 'react-native';
+
+import { StrictTheme } from '../../../theme';
+import { DefaultContentStyle } from './types';
+
+export const getDefaultStyle = ({
+  tokens: { space },
+}: StrictTheme): Required<DefaultContentStyle> =>
+  StyleSheet.create({
+    buttonPrimary: {
+      marginVertical: space.medium,
+    },
+    buttonPrimaryLabel: {}, // themed value only
+    buttonSecondary: {
+      marginVertical: space.xs,
+      minWidth: '50%',
```

Comment:
This is to allow the secondary buttons to "stack" if scenarios where button labels are long-ish